js/apps/keycloak-server/scripts/start-server.js

```diff
@@ -21,11 +23,15 @@ const AUTH_RETRY_LIMIT = 3;
 await startServer();
 
 async function startServer() {
-  await downloadServer();
+  var local = false;
+  if (process.argv[2] && process.argv[2] === "--local") {
```

Comment:
I think now is a good time to clean up the code to parse command line arguments. For this we can use [`util.parseArgs()`](https://nodejs.org/api/util.html#utilparseargsconfig). For more information see:

- https://exploringjs.com/nodejs-shell-scripting/ch_node-util-parseargs.html

Reply:
I have utilized the above tool to create a method which I think should adequately cover the use cases in this script and allow for future modifications without too much headache!